Data and scripts for reproducing the analyses in KK lab 

This repository contains general workflow of the shotgun metagenomic analysis conducted in the lab 

##Directory Structure envs: Miniconda used for this project scripts: folder containing code used for this project (they use jupyter and Rmarkdown so we shall see what we used) tables: input data used - Metadata, Metagenomics raw data- relative abundance data obtained from GTDB megan output.

Software versions: Python, R version, tidyversion etccc

library ( you mean reference database that we used?)

Raw data availability

raw sequencing data has been uploaded to Staff3
assemblies and annotations will be available here
Citation Please cite on our manuscript on

Contact Please diret any questions or feedback to Dayang Asyiqin and Kazuyuki Kasahara

Figures to script relationship
